MOBILE_API.xml

```diff
@@ -3328,6 +3428,7 @@
     
     <struct name="ClimateControlCapabilities" since="4.5">
         <description>Contains information about a climate control module's capabilities.</description>
+        <!-- need an ID in the future -->
```

Comment:
Please remove this

Reply:
fixed on a8b4b1930782409dc886831843c72e223a77bcaa